tests/other/test_smtlibv2.py

```diff
@@ -11,15 +12,55 @@
     arithmetic_simplify,
     constant_folder,
     replace,
+    BitVecConstant,
 )
 from manticore.core.smtlib.solver import Z3Solver
 from manticore.core.smtlib.expression import *
 from manticore.utils.helpers import pickle_dumps
+from manticore import config
 
 # logging.basicConfig(filename = "test.log",
 #                format = "%(asctime)s: %(name)s:%(levelname)s: %(message)s",
 #                level = logging.DEBUG)
 
+DIRPATH = os.path.dirname(__file__)
+
+
+class RegressionTest(unittest.TestCase):
+    def test_related_to(self):
+        import gzip
+        import pickle, sys
+
+        filename = os.path.abspath(os.path.join(DIRPATH, "data", "ErrRelated.pkl.gz"))
```

Comment:
What's actually in the `ErrRelated.pkl.gz` file? How do I create it?

Reply:
Good to know. I think we should put something like that in a comment for this test, just to let someone else know what it means.